### PR TITLE
Move project published text to Text

### PR DESCRIPTION
**Pull request by SMaxOwok**

### CODE REVIEW

File: api/db/migrate/20190122160027_move_published_text_from_projects_to_texts.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        WHERE id IN (SELECT published_text_id FROM projects)
+    SQL
+
+    remove_belongs_to :projects, :published_text, type: :uuid
+  end
+
+  def down
+    add_belongs_to :projects, :published_text, type: :uuid
+
+    execute <<-SQL.squish
```
Comment (SMaxOwok) [inline api/db/migrate/20190122160027_move_published_text_from_projects_to_texts.rb]: Can someone test this?

I first rolled back without the `INNER JOIN` so my data got into a weird state with mismatched texts.  It looked like it worked, but I just want to be sure with valid data.

Comment (SMaxOwok) [reply api/db/migrate/20190122160027_move_published_text_from_projects_to_texts.rb]: Or maybe @scryptmouse can tell by just looking at it.

**Comment by zdavis** [comment]
Please update the migration time stamps and I will test again.

File: api/db/migrate/20190122160027_move_published_text_from_projects_to_texts.rb (hunk 1 above)
Comment (zdavis) [reply api/db/migrate/20190122160027_move_published_text_from_projects_to_texts.rb]: Tested. Works.

**Pull request merged**